src/Crash/CrashTests.cpp

```
@@ -0,0 +1,231 @@
+				(void)hasData;
+			}
+
+			// Helper to corrupt PlayerCharacter singleton
+			__declspec(noinline) void CauseCorruptedPlayerSingleton()
+			{
+				// Get the real PlayerCharacter singleton
+				auto player = RE::PlayerCharacter::GetSingleton();
+				if (!player) {
+					// If we can't get player, just cause a null dereference anyway
+					RE::PlayerCharacter* nullPlayer = nullptr;
+					volatile auto formID = nullPlayer->GetFormID();
```

Cppcheck (2.19.0)

[error] Null pointer dereference

(nullPointer)